src/app/static/src/app/root.ts

```diff
@@ -37,6 +38,7 @@ export interface RootState extends TranslatableState {
     adrDatasets: any[],
     adrKey: string | null,
     adrKeyError: Error | null,
+    adrSchemas: ADRSchemas | null,
```

Comment:
This is starting to feel like it might belong in an adr module rather than just on root state, what do you think? Will there be any more bits still to come on adr?

Reply:
Pretty sure this is the last one. Have considered same, but would it make the code simpler or more complicated?

Reply:
Yeah, it's a bit borderline. Fine as it is I think, but probably worth changing if the adr properties do expand any more..